clang/include/clang/Basic/ObjCRuntime.h

```diff
@@ -101,10 +101,17 @@ class ObjCRuntime {
     // The GNUstep runtime uses a newer dispatch method by default from
     // version 1.6 onwards
     if (getKind() == GNUstep && getVersion() >= VersionTuple(1, 6)) {
-      if (Arch == llvm::Triple::arm ||
-          Arch == llvm::Triple::x86 ||
-          Arch == llvm::Triple::x86_64)
+      switch (Arch) {
```

Comment:
The version check above is not quite right for these.

AArch64 support is in 1.9.
RV64 in 2.2.

I think mips64 is correct (it looks like I added the code 9 years ago, which probably lines up with the 1.6 release).

Reply:
> The version check above is not quite right for these.

This was my first approach, but it seems like GNUstep and libobjc2 unit tests are always emitting -fobjc-runtime=gnustep-2.0

So one would need to change this flag in the build configuration everytime libobjc2 gets updated.

Reply:
I can do the version check properly and update the libobjc2 README to properly document this. I probably need to rewrite the detection in gnustep-make too -_-

Reply:
> This was my first approach, but it seems like GNUstep and libobjc2 unit tests are always emitting -fobjc-runtime=gnustep-2.0

That's fine.  We should bump that in both places to the correct version.  It's better than having the compiler emit things that are not supported when targeting a specific version of the runtime.